gateway/Caddyfile

```diff
@@ -0,0 +1,50 @@
+http://:3000 {
+	log {
+		level INFO
+	}
+
+	# CORS
+
+	@OPTIONS {
+		method OPTIONS
+	}
+	header {
+		Access-Control-Allow-Origin {$CLIENT_URL}
+		Access-Control-Allow-Credentials true
+		Access-Control-Allow-Headers RSESSID,Content-Type
+		defer
+	}
+	handle @OPTIONS {
+		respond 204
+	}
+
+	# Unprotected routes
+
+	handle /users {
+		reverse_proxy http://agartex-authentication:3100
+	}
+
+	handle /sessions {
+		reverse_proxy http://agartex-authentication:3100
+	}
+
+	# Protected routes
+
+	handle * {
+		forward_auth http://agartex-authentication:3100 {
+			uri /users
```

Comment:
https://github.com/Agar-OSS/agartex-authentication/pull/8
This needs to now be `/sessions`